eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/SubmittedVariant.java

```diff
@@ -37,8 +38,11 @@
 
     private boolean supportedByEvidence;
 
+    private LocalDateTime createdDate;
+
     public SubmittedVariant(String assemblyAccession, int taxonomyAccession, String projectAccession, String contig,
-                            long start, String referenceAllele, String alternateAllele, boolean supportedByEvidence) {
+                            long start, String referenceAllele, String alternateAllele, boolean supportedByEvidence,
+                            LocalDateTime createdDate) {
```

Comment:
It looks that so far createdDate is always initialized as null. If there is still code to be added and that parameter is going to be used, I would leave it in the constructor. If not, I would remove it from the constructor, even if we keep getCreatedDate in the class just to implement the interface.

Reply:
yeah, I was not convinced about this either. the class needs to have this field to be compliant with the interface, but maybe the setter is enough.